predict.py

```diff
@@ -51,6 +51,7 @@ def setup(self):
     def predict(
         self,
         prompt: str = Input(description="Input prompt", default=""),
+        negative_prompt: str = Input(description="Input negative prompt", default=""),
```

Comment:
suggestion: change to `negative_prompt: str = Input(description="Input negative prompt", default=None)` since it is optional

Reply:
Thank you, @chenxwh! Updated!